web/src/pages/Consumer/locales/zh-CN.ts

```diff
@@ -17,8 +17,8 @@
 export default {
   'page.consumer.form.itemRuleMessage.username':
     '最大长度100，仅支持字母、数字和 _ ，且只能以字母开头',
-  'page.consumer.form.itemExtraMessage.username': '用户名需唯一',
+  'page.consumer.form.itemExtraMessage.username': '名称需唯一',
```

Comment:
@liuxiran Other resources also use `名称`, not `路由名称` or `服务名称`. So shall we follow that practice here?

Reply:
@imjoey I also noticed that, the lable name `page.consumer.username` would be better to follow other moudules.

for the `page.consumer.form.itemExtraMessage.username`, which is used to show a input tip
![2021-01-05 16-50-53屏幕截图](https://user-images.githubusercontent.com/2561857/103625986-946f7600-4f76-11eb-8961-3f4d0de697e2.png)
IMHO, It may be better to be specific. 
meanwhile, the itemExtraMessage are not yet unified in different modules, maybe we need to  deal with it uniformly